static/src/javascripts/projects/commercial/modules/comment-adverts.ts

```diff
@@ -12,10 +12,25 @@ import { getAdvertById } from './dfp/get-advert-by-id';
 import { refreshAdvert } from './dfp/load-advert';
 
 const createCommentSlot = (canBeDmpu: boolean): HTMLElement => {
-	const sizes: SizeMapping = canBeDmpu
-		? { desktop: [adSizes.halfPage, adSizes.skyscraper] }
-		: {};
-	const adSlot = createAdSlot('comments', { sizes });
+	const adSlot = createAdSlot('comments');
+
+	if (!canBeDmpu) {
```

Comment:
It seemed to make more sense to keep the DCR size mapping for comments and remove rather than add the DMPU, this code is frontend rendered articles only anyway.